src/main/java/wolox/training/models/Book

```diff
@@ -4,50 +4,53 @@ import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
 
 @Entity
 public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
 
-    @Column(nullable = false)
+    @NotNull
     private String title;
 
-    @Column(nullable = false)
+    @NotNull
     private String author;
 
     @Column(nullable = true)
```

Comment:
You can remove this annotation.

Reply:
ok, I have a question about that, if I remove it, the default would be to allow  "nullable" to be true?